merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -502,6 +502,10 @@ private List<ParameterValidationRecord> getExportValidations(
     for (final var element : exportTypeElement.getEnclosedElements()) {
       if (element.getAnnotation(Export.Validation.class) == null) continue;
 
+      // Is there a way to tie this to the actual class rather than the string?
+      boolean isSimpleValidation = !(element.getKind() == ElementKind.METHOD &&
+          ((ExecutableElement) element).getReturnType().toString().equals("gov.nasa.jpl.aerie.contrib.models.ValidationResult"));
```

Comment:
@mattdailis or @Twisol might know some fancier way, but if you want to tie it to the class, you could do this:
```suggestion
      // Is there a way to tie this to the actual class rather than the string?
      boolean isSimpleValidation = !(element.getKind() == ElementKind.METHOD &&
          ((ExecutableElement) element).getReturnType().toString().equals(ValidationResult.class.getTypeName()));
```

Reply:
Awesome, this is exactly what I was looking for!